templates/terraform/examples/region_network_endpoint_group_appengine.tf.erb

```diff
@@ -0,0 +1,69 @@
+// App Engine Example
+resource "google_compute_region_network_endpoint_group" "<%= ctx[:primary_resource_id] %>" {

```

Comment:
can you add `provider = google-beta` to each of these resources? That's needed to get the generated tests to run.